Source/Orts.Simulation/Simulation/RollingStocks/MSTSWagon.cs

```diff
@@ -148,7 +149,8 @@ public enum WindowState
         public float TrailLocoResistanceFactor; // Factor to reduce base and wind resistance if locomotive is not leading - based upon original Davis drag coefficients
 
         bool TenderWeightInitialize = true;
```

Comment:
I know this is not part of your commit, but perhaps the sense could be reversed from:

bool TenderWeightInitialize = true;

to

bool TenderWeightInitialized = false;

Elsewhere in Open Rails, "Initialised" and "Initialized" are used, so using the opposite is confusing.

Reply:
Ok, done.